src/doc/using_visit/VisualizationWindows/Interactor_settings.rst

```diff
@@ -21,47 +18,27 @@ that have interactor properties. The **Interactors window** is shown in
+If you want VisIt to always force a square zoom rectangle so that you don't have to use the *Shift* key, you can click on the **Clamp to Square** check box, click **Apply** in the **Interactors window** and save your settings.
 
-Using the **Clamp to Square** zoom mode is a good way to maximize the
-amount of the vis window that is used when you zoom in on plots and when
-the vis window is in zoom mode. When the vis window is in navigate mode,
-the middle mouse button also effects a zoom. By default, zooming with
-the middle mouse button zooms into the plots but keeps the same vis
-window viewport which may, depending on the aspect ratio of the plots,
-not make the best use of the vis window's pixels. Fortunately, you can
-turn on the **Fill viewport on zoom** check box to force middle mouse
-zooming to also enlarge the viewport to its largest possible size in
-order to make better use of the vis window's pixels.
+Using the **Clamp to Square** zoom mode is a good way to maximize the amount of the vis window that is used when you zoom in on plots and when the vis window is in zoom mode. 
+When the vis window is in navigate mode, the middle mouse button also effects a zoom. 
+By default, zooming with the middle mouse button zooms into the plots but keeps the same vis window viewport which may, depending on the aspect ratio of the plots, not make the best use of the vis window's pixels. 
+Fortunately, you can turn on the **Fill viewport on zoom** check box to force middle mouse zooming to also enlarge the viewport to its largest possible size in order to make better use of the vis window's pixels.
 
 Navigation styles
 ~~~~~~~~~~~~~~~~~
 
-When VisIt displays 3D plots, there are a few navigation styles from
-which you can choose by clicking on the following radio buttons in the
-**Interactors window**: **Trackball**, **Dolly**, and **Flythrough**.
-The default navigation style for 3D plots is: Trackball and it allows
-you to interactively rotate plots and move around them but it keeps the
-camera at a fixed distance from the plots and while it can get infinitely
-close to plots when you zoom in, it can never touch them or go inside of
-them. The Dolly navigation style behaves like the trackball style except
-that the when the camera zooms, it is actually moved. The Flythrough
-navigation style moves the camera and allows you to fly into plots and
-out the other side.
+When VisIt displays 3D plots, there are a few navigation styles from which you can choose by clicking on the following radio buttons in the **Interactors window**: **Trackball**, **Dolly**, and **Flythrough**.  
+The default navigation style for 3D plots is: Trackball and it allows you to interactively rotate plots and move around them but it keeps the camera at a fixed distance from the plots and while it can get infinitely close to plots when you zoom in, it can never touch them or go inside of them. 
```

Comment:
```suggestion
The default navigation style for 3D plots is **Trackball**. This allows users to interactively rotate plots and move around them but it keeps the camera at a fixed distance from the plots. While it can get infinitely close to plots when zoomed in, it can never touch them or go inside of them. 
```